src/components/DropZone/DropZone.tsx

```diff
@@ -304,30 +307,58 @@ export class DropZone extends React.Component<CombinedProps, State> {
 
     this.setState({error: this.props.error});
 
-    if (!this.dropNode) {
+    const dropNode: HTMLDivElement | Document | null =
+      'current' in this.dropNode ? this.dropNode.current : document;
```

Comment:
Could we change this to
```tsx
const dropNode = this.dropNode.current || document;
```

Reply:
`[ts]
Property 'current' does not exist on type 'RefObject<HTMLDivElement> | Document'.
  Property 'current' does not exist on type 'Document'.`

I think that a type check seems to be imperative in this case, before accessing the current property